include/coloring/DRVertexQueue.hpp

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Consider git mv'ing it to include/structures and parameterizing it by <A, B> (instead of int, int) - of course this would mean that the whole class goes to hpp file.

Reply:
On a second thought, please see how Aux::BucketPQ structure is used in GreedyVertexColoring.cpp - I find this on the first sight to be completely redundant... and if not, please explain the difference

Reply:
Yes, that is exactly the same structure - replacing with BucketPQ